rclcpp/include/rclcpp/experimental/intra_process_manager.hpp

```diff
@@ -389,6 +408,7 @@ class IntraProcessManager
       if (subscription_it == subscriptions_.end()) {
         throw std::runtime_error("subscription has unexpectedly gone out of scope");
       }
+<<<<<<< HEAD
```

Comment:
It looks like there was a conflict that mergify didn't handle.

Reply:
thanks, I lost track of this PR.
https://github.com/ros2/rclcpp/pull/1645/commits/73b1b0e947ff03a3dbcecdf267f39880d54cb21b should've solved the conflicts.